### PR TITLE
Update to 1.19

### PR DESCRIPTION
**Pull request by Arathain**
Feel free to make sure everything's nice & tidy before merging - I don't know if the buildscript is 100% functional in terms of publishing

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -52,19 +51,25 @@ repositories {
             includeGroup 'com.terraformersmc'
         }
     }
+    maven {
+        url "https://plugins.gradle.org/m2/"
+    }
```
Comment (Pyrofab) [inline build.gradle]: Is this maven necessary ?

File: src/main/java/io/github/ladysnake/creeperspores/common/CreeperlingEntity.java [hunk 2]
```diff
@@ -228,7 +230,7 @@ public float getPathfindingFavor(BlockPos pos, WorldView worldView) {
         // Creeperlings like sunlight
         int skyLightLevel = worldView.getLightLevel(LightType.SKY, pos);
         // method_28516 == getBrightness
-        float skyFavor = worldView.getDimension().getBrightness(skyLightLevel) * 3.0F;
+        float skyFavor = worldView.getDimension().getSkyAngle(skyLightLevel) * 3.0F;
```
Comment (Pyrofab) [inline src/main/java/io/github/ladysnake/creeperspores/common/CreeperlingEntity.java]: Huh. Is this really equivalent ?

File: settings.gradle [hunk 3]
```diff
@@ -7,13 +7,16 @@ pluginManagement {
...
+            url = 'https://maven.quiltmc.org/repository/release'
         }
         maven {
             name = 'Cotton'
             url = 'https://server.bbkr.space/artifactory/libs-release/'
         }
         gradlePluginPortal()
+        maven {
+            url "https://plugins.gradle.org/m2/"
+        }
```
Comment (Pyrofab) [inline settings.gradle]: Shouldn't this be equivalent to `gradlePluginPortal` ?

**Review by Pyrofab — COMMENTED** [review]
Thanks for the work ! Just wanted to confirm a few things.

File: src/main/java/io/github/ladysnake/creeperspores/common/CreeperlingEntity.java (hunk 2 above)
Comment (Arathain) [reply src/main/java/io/github/ladysnake/creeperspores/common/CreeperlingEntity.java]: Turns out it wasn't - fixed in latest commit

File: src/main/java/io/github/ladysnake/creeperspores/common/CreeperlingEntity.java [hunk 4]
```diff
@@ -230,11 +231,11 @@ public float getPathfindingFavor(BlockPos pos, WorldView worldView) {
         // Creeperlings like sunlight
         int skyLightLevel = worldView.getLightLevel(LightType.SKY, pos);
         // method_28516 == getBrightness
-        float skyFavor = worldView.getDimension().getSkyAngle(skyLightLevel) * 3.0F;
+        float skyFavor = computeBrightnessByLightLevel(worldView.getDimension().ambientLight())[skyLightLevel] * 3.0F;
         // But they can do with artificial light if there is not anything better
-        // One day we will get in trouble for using this method, but in the mean time it's used by everything else
-        @SuppressWarnings("deprecation") float brightnessAtPos = worldView.getBrightness(pos);
-        float favor = Math.max(brightnessAtPos - 0.5F, skyFavor);
+        // One day we will get in trouble for using this method, but in the meantime it's used by everything else
```
Comment (Pyrofab) [inline src/main/java/io/github/ladysnake/creeperspores/common/CreeperlingEntity.java]: If the method isn't deprecated you can probably remove this comment

File: src/main/resources/quilt.mod.json [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+      "name": "Creeper Spores",
+      "description": "Creeper Spores !",
+      "contact": {
+        "sources": "https://github.com/Ladysnake/Creeper-Spores",
+        "issues": "https://github.com/Ladysnake/Creeper-Spores/issues",
+        "discord": "ladysnake.glitch.me/discord",
+        "homepage": "https://ladysnake.github.io/wiki/creeper-spores"
+      },
+      "icon": "assets/creeperspores/textures/logo.png"
+    },
```
Comment (Pyrofab) [inline src/main/resources/quilt.mod.json]: Wait where did the license and authors field go ?

**Pull request merged**